css/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+    background: #FFF;
+}
+
+.page-header {
+    background: #FFF;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.header-icon {
+    width: 151px;
+    height: 101px;
+    flex-shrink: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
+}
+
+.nav-list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-item {
+    display: block;
+    padding: 10px;
+    text-decoration: none;
+    color: #957554;
+    list-style: none;
+    font-family: Ubuntu;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+}
+
+.header-loop-icon {
+    width: 16px;
+    height: 16px;
+    flex-shrink: 0;
+    background: #FFF;
+    border: 0;
+}
+
+.drop-down {
+    background: #111;
+}
```

Comment:
in general it is better to do like in video
https://streamable.com/tlia0d

and screens for convenience
https://monosnap.com/list/65048ab41f56780e32a54da7